updates/2016-05-11.yml

```diff
@@ -0,0 +1,8 @@
+added:
```

Comment:
remove this for now, the doc isnt public so we dont need to push an update. we will broadcast and update once the doc is public.